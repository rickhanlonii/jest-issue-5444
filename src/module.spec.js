const m = require("./module");

describe("EventEmitter", () => {
  it("should always update snapshot to now", () => {
    expect(m).toMatchSnapshot();
  });
});
