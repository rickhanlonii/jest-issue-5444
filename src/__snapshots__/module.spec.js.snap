// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EventEmitter should always update snapshot to now 1`] = `
Object {
  "answer": 42,
}
`;
